Plugins/Onnx/include/Acts/Plugins/Onnx/OnnxRuntimeBase.hpp

```diff
@@ -49,6 +49,12 @@ class OnnxRuntimeBase {
   std::vector<std::vector<float>> runONNXInference(
       NetworkBatchInput& inputTensorValues) const;
 
+  std::vector<std::vector<std::vector<float>>> runONNXInferenceMultiOutput(
+      NetworkBatchInput& inputTensorValues) const;
+
+  std::vector<std::vector<float>> runONNXInferenceMultiOutput(
+      std::vector<float>& inputTensorValues) const;
+
```

Comment:
The equivalent only really exist because of a piece of legacy code somewhere (the MLTrackClassifier) that needs it (and because I didn't want to change that code). It is really inefficient to do the inference that way so I wouldn't even create this function.

Reply:
Yeah I agree actually. Will remove.